src/components/Request/Request.react.js

```diff
@@ -52,19 +53,27 @@ export default class Request extends Component {
       .replace(/\s+|\n\r|\n|\r$/gm, '');
     pubkey = encodeURIComponent(pubkey);
     
-    // URI for service
-    this.baseRequestUri = "meta://information?service=" + this.props.service;
+    // URI for usage
+    this.baseRequestUri = "meta://information?usage=" + this.props.usage;
+
     // URI for request
-    this.props.request.map((req) => {this.baseRequestUri += "&request=" + req;});
+    this.baseRequestUri += "&request=";
+    this.props.request.map((req) => { this.baseRequestUri += req + ',' });
```

Comment:
`Array.join` is better than `map`. At next PR, please apply like this
`this.baseRequestUri += this.props.request.join(',')`